packages/flutter_tools/lib/src/base/utils.dart

```diff
@@ -89,8 +91,13 @@ String getElapsedAsMilliseconds(Duration duration) {
 }
 
 /// Return a String - with units - for the size in MB of the given number of bytes.
-String getSizeAsMB(int bytesLength) {
-  return '${(bytesLength / (1024 * 1024)).toStringAsFixed(1)}MB';
+String getSizeAsMB(int bytesLength, {
```

Comment:
Should we rename this to `getSizeAsPlatformMB`? I don't feel strongly

Reply:
Sure, renamed.